tensorflow_datasets/core/features/image_feature.py

```diff
@@ -45,6 +44,13 @@
     'jpeg': [tf.uint8],
 }
 
+_COLORS = [
+    '#AA0DFE', '#3283FE', '#85660D', '#782AB6', '#565656', '#1C8356', '#16FF32',
```

Comment:
I liked your previous idea of using matplotlib for the colormap. How about using the matplotlib API to generate the colormap ?

https://matplotlib.org/3.1.0/tutorials/colors/colormap-manipulation.html

Something like

```python
@utils.memoize()  # Memoize to only be computed once
def _get_colormap() -> np.ndarray:
  """Returns the `np.array(shape=(255, 3), dtype=uint8)` RGB colors."""
```

Reply:
And the you can simply use something like:

```python
def _apply_colormap(img: np.ndarray) -> np.ndarray:
  img = img.squeeze(axis=-1)
  cmap = _get_colormap()  # (255, 3) id->color mapping 
  return cmap[img]  # Apply mapping from (x, y) -> (x, y, 3)
```

Reply:
Have updated the code, to align with your expectations. Let me know if I have used the decorator properly.